lib/medtronic/processData.js

```diff
@@ -451,7 +459,94 @@ function buildSuspendResumeRecords(records) {
   return postrecords;
 }
 
-function buildBasalProfiles(records) {
+function getCarbRatios(encoded, units) {
+  var carbRatios = [];
+  for(var j = 0; j < 8; j++ ) {
+    var carbRatio = struct.unpack(encoded,(j*3)+2,'bS',['offset', 'ratio']);
+    if (carbRatio.offset === 0 && carbRatio.ratio === 0) {
+      break;
+    }
+    var startTime = carbRatio.offset * 30 * sundial.MIN_TO_MSEC;
+    if(units === 'exchanges') {
+      carbRatio.ratio /= 100.0;
+    }
+    carbRatios.push( { start: startTime, amount: (carbRatio.ratio / 10.0).toFixedNumber(5) } );
+  };
+
+  return carbRatios;
+}
+
+function getBGTargets(encoded, units) {
+  var bgTargets = [];
+  for(var j = 0; j < 8; j++ ) {
+    var bgTarget = struct.unpack(encoded,(j*3)+1,'bbb',['offset', 'low','high']);
+    if (bgTarget.offset === 0 && bgTarget.low === 0 && bgTarget.high === 0) {
+      break;
+    }
+    var startTime = bgTarget.offset * 30 * sundial.MIN_TO_MSEC;
+    if(units === 'mmol/L') {
+      bgTarget.low /= 10.0;
```

Comment:
It actually took me a second to understand the `/=` shorthand here. I've rarely seen this one. Do we use it elsewhere in drivers? If not, perhaps consider the longer version for consistency/clarity?